content/guides/references/migration-guide.md

```diff
@@ -117,6 +117,178 @@ cy.intercept('GET', '/projects', {
 
 Unlike `cy.route()`, `cy.intercept()` currently does _not_ allow you to override a previous response. For more information on this, see [#9302](https://github.com/cypress-io/cypress/issues/9302) and [this blog post](https://glebbahmutov.com/blog/cypress-intercept-problems/#no-overwriting-interceptors). Overriding responses will be added in a future release.
 
+## Migrating to Component Testing 7.0
+
+### cypress.json
+
+Remove `"experimentalComponentTesting": true` since the flag will give you an error.
+
+Remember that from 7.0 you can configure `cypress.json` and have specific flags for component testing or e2e.
+
+For example if you want video recording enabled only for End-to-end testing, you can disbale it for compoent testing like so
+
+```json
+{
+  "testFiles": "**/*spec.{js,jsx,ts,tsx}",
+  "video": true,
+  "component": {
+    "componentFolder": "src/components",
+    "video": false
+  }
+}
+```
+
+[Read the docs for the new config](/guides/references/configuration)
+
+### Plugins file
```

Comment:
Convention in the docs seems to be title case. So "Plugins File".

You will want to apply this convention to all headings.